src/packages/core/src/services/data.ts

```diff
@@ -68,8 +68,16 @@ export default class DataService {
     ) {
       await this.requestWithFilters(filters, this.widget.attributes?.widgetConfig?.paramsConfig);
     } else {
-      this.setEditor({ widgetData: null });
+      this.setEditor({
+        widgetData: null,
+        advanced: !this.widget?.attributes?.widgetConfig?.paramsConfig
```

Comment:
This fixes an issue in this case: 

You load a advanced widget on initial load, if we dont toggle this property "advanced" true|false the previous advanced widget will show in the UI 

So now when you restore a widget we get the correct widget if initial was advanced.